spec/acceptance/suites/default/01_simp_server_bootstrap_spec.rb

```diff
@@ -55,13 +55,13 @@ class { 'pupmod::master':
 
   context 'bootstrap simp server' do
     it 'should set up a simp server' do
-       apply_manifest_on(master, master_manifest, :accept_all_exit_codes => true)
-       apply_manifest_on(master, master_manifest, :accept_all_exit_codes => true)
+       apply_manifest_on(master, master_manifest, accept_all_exit_codes: true)
```

Comment:
Please don't change this. We're still going back and forth on that and, at last discussion, were sticking with the main syntax since you can't use the new one for everything.